### PR TITLE
05 03 caps update

### PR DESCRIPTION
**Pull request by reem**
Adds 05-03-2023 caps update in accordance with https://governance.aave.com/t/arc-gauntlet-recommendations-for-polygon-v3-and-arbitrum-v3/12919

IPFS Preview: https://app.aave.com/governance/ipfs-preview/?ipfsHash=0x5ef5fe9fcc8b8f5748b49dce1260e48e95065526f3a1b36752f068addc08715c

Jira: https://gauntlet-network.atlassian.net/browse/RR-124

### CODE REVIEW

File: diffs/preTestArbitrumUpdate20230503_postTestArbitrumUpdate20230503.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Reserve changes
+
+### Reserves altered
+
+#### EURS ([0xD22a58f79e9481D1a88e00c343885A588b34b68B](https://https://arbiscan.io/address/0xD22a58f79e9481D1a88e00c343885A588b34b68B))
+
+| description | value before | value after |
+| --- | --- | --- |
+| supplyCap | 60,000 EURS | 65,000 EURS |
+| borrowCap | 45,000 EURS | 65,000 EURS |
```
Comment (reichiang) [inline diffs/preTestArbitrumUpdate20230503_postTestArbitrumUpdate20230503.md:10]: isn't current borrow cap 40,000?

Comment (reichiang) [reply diffs/preTestArbitrumUpdate20230503_postTestArbitrumUpdate20230503.md:10]: nvm, looks like 45,000 on-chain - do we update the text in forum post in this case?

File: specs/description20230503.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- BAL supply (92.4%) and borrow (76%) caps on Polygon v3.
+- EURS supply (78.4%) and borrow (78%) on Arbitrum v3.
+
+In accordance to our Borrow and Supply Cap methodology, Gauntlet would like to raise the caps of the above assets to the following.
+
+
+#### Arbitrum v3
+|      | Current Supply Cap | Recommended Supply Cap | Current Borrow Cap | Recommended Borrow Cap |
+| ---- | ------------------ | ---------------------- | ------------------ | ---------------------- |
+| EURS | 60,000              | 65,000                  | 40,000              | 65,000                  |
```
Comment (reichiang) [inline specs/description20230503.md:21]: Do we want to update Current Borrow Cap to 45,000 here?

**Review by reichiang — APPROVED** [review]
lgtm

**Review by jmomx — APPROVED** [review]
cap change looks good to me - seems like there might be a typo in the original post as rei mentioned but other than that i dont see any issues

**Comment by reem** [comment]
Tenderly sim: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/391fe231-4e93-4faf-9721-74188de51ffc

**Comment by reem** [comment]
Submitted to safe.

**Comment by reem** [comment]
Executed

**Pull request closed without merging**